R/extremes-data.R

```diff
@@ -210,7 +212,27 @@ createDataRows <-
         {
           relatedValue <- relatedSet$value
         }
+        
+        if (!isEmptyOrBlank(x$relatedPrimary)) {
+          if (nrow(x$relatedPrimary) > nrow(x$points) ||
+              nrow(x$points) > nrow(x$relatedPrimary)) {
```

Comment:
Like (if(nrow(x$relatedPrimary) != nrow(x$points){ blah }
       if(nrow(x$relatedUpchain) != nrow(x$points))
Sorry for the confusion, I don't have any secret R magic to get rid of the duplication off the top of my head.